frontend/src/domains/plays/PlayDetails/PlayDetails.js

```diff
@@ -1,34 +1,118 @@
-  
-export default function Play() {
+import { getLocaleDate } from '../../../utilities/getLocaleDate'
+import ArrowBackIcon from '../../../components/icons/ArrowBackIcon'
+import Icon from '../../../utilities/Icon' 
+import PlayRanking from '../../../components/PlayRanking/PlayRanking'
+import Star from '../../../components/icons/StarIcon'
+
+export default function PlayDetails() {
   const params = useParams()
-  const [play, setPlay] = useState({})
+  const [play, setPlay] = useState({ players: [] })
 
   useEffect(() => {
     fetchPlayDetails(params.id).then(setPlay)
   }, [])
 
   return (
-    <StyledPlay>
-      <p><StyledImage src={apiBaseURL + play.imageURL} alt="No image" /></p>
-      <h2>Game</h2><p>{play.gameTitle}</p>
-      <h2>Date</h2><p>{play.playDate}</p>
-      <h2>Players</h2><p>{play.players}</p>
-      <h2>Playing Time</h2><p>{play.playingTime}</p>
-      <h2>Rating</h2><p>{play.playRating}</p>
-    </StyledPlay>
+    <>
+      <StyledLink to="/log">
+        <ArrowBackIcon />
+      </StyledLink>
+      <h1>{play.gameTitle}</h1>
+      <h2>Details</h2>
+      <StyledPlay>
+        <StyledImage>
+          <img src={apiBaseURL + play.imageURL} alt="No photo available" />
+        </StyledImage>
+        <h4>{getLocaleDate(play.playDate)}</h4>
+        <StyledOverview>
+          <StyledContainer>
+            <Icon type="group" alt="" />
+            <StyledCaption>Players</StyledCaption>
+            <p>{play.players.length}</p>
+          </StyledContainer>
+          <StyledContainer>
+            <Icon type="access_time" alt="" />
+            <StyledCaption>Playing time (min)</StyledCaption>
+            <p>{play.playingTime}</p>
+          </StyledContainer>
+          <StyledContainer>
+            <Star isSelected={true} />
+            <StyledCaption>Rating</StyledCaption>
+            <p>{play.playRating}</p>
+          </StyledContainer>
+        </StyledOverview>
+      </StyledPlay>
+      <PlayRanking play={play}/>
+    </>
   )
 }
 
+const StyledLink = styled(Link)`
+  text-decoration: none;
+`
+
 const StyledPlay = styled.div`
   margin: 0;
-  padding: 20px 0;
+  padding: 0;
+
+  & h4, & p {
+    text-align: center;
+  }
+`
+
+const StyledImage = styled.div`
+  background: var(--inner-shadow-dark);
+  height: 188px;
+  margin-top: 25px;
+  width: 300px;
+
+  & img {
+    align-items: center;
+    color: var(--text-color);
+    display: flex;
+    font-family: var(--caption-font);
+    font-size: 12px;
+    height: 188px;
+    justify-content: center;
+    margin: 0;
+    object-fit: cover;
+    text-align: center;
+    width: 300px;
+  }
+` 
+
+const StyledOverview = styled.div`
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: 1fr 1fr 1fr;
+  margin: 25px 0;
+  text-align: center;
+  word-break: normal;
+  
+  & span {
+    color: var(--primary);
+  }
 `
 
-const StyledImage = styled.img`
-  max-width: 100%;
+const StyledCaption = styled.p`
```

Comment:
Falls du caption öfter als nur hier benutzt, könntest du auch hier eine class in den Global Styles definieren '.caption' und die dann universal einsetzen. Hat dann auch gleich den Vorteil, dass die Caption immer gleich aussieht :)

Reply:
Danke für den Tipp!